controllers/slicegateway/slicegateway.go

```diff
@@ -30,6 +30,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
We dont really need this. You can access the logger from the reconcile object: r.Log

Reply:
You are right. I missed using the `r.Log` after making `createPodDisruptionBudgetForSliceGatewayPods` a method on `SliceGwReconciler`. Done now. Thanks!